corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -104,8 +106,8 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
 
         if (menuResponse.breadcrumbs) {
             menusUtils.showBreadcrumbs(menuResponse.breadcrumbs);
-            if (menuResponse.langs && menuResponse.langs.length > 1 && !appPreview && changeFormLanguage) {
-                menusUtils.showLanguageMenu(menuResponse.langs);
+            if (!appPreview && ((menuResponse.langs && menuResponse.langs.length > 1 && changeFormLanguage) || enablePrintOption)) {
+                menusUtils.showFormMenu(menuResponse.langs, changeFormLanguage);
```

Comment:
Should the higher level "enableLanguageMenu" be `menuResponse.langs && menuResponse.langs.length > 1 && changeFormLanguage)` ? 

As in, we wouldn't want to show the language option view if `menuResponse.langs && menuResponse.langs.length > 1` is False even if `changeFormLanguage` is True. That's how i'm interpreting the original behaviour

Reply:
> Should the higher level "enableLanguageMenu" be menuResponse.langs && menuResponse.langs.length > 1 && changeFormLanguage) ?
there is a check in `showFormMenu` for langs and langs is null, then the language menu will not be added. So these would result in the equivalent behavior